src/main/scala/org/scalajs/tools/tsimporter/parser/TSDefParser.scala

```diff
@@ -37,7 +37,7 @@ class TSDefParser extends StdTokenParsers with ImplicitConversions {
 
       // Future reserved keywords in Strict mode - some used in TypeScript
       "implements", "interface", "let", "package", "private", "protected",
-      "public", "static", "yield",
+      "public", "static", "yield", "from",
```

Comment:
`from` is not a keyword. The `import` statements recognize `from` as a special identifier, but only in that specific context. In all non-import contexts, `from` should stay a regular identifier, otherwise `var from: number` will not be valid anymore.

Reply:
Ok I see. Is `lexical.Identifier("from")` the right way to parse it in the import context?

Reply:
I am not sure what is the best way to go about this. `accept`+`lexical.Identifier("from")` might be a reasonable choice.